### PR TITLE
refactor(stepper): Figma match and Tailwindify

### PR DESCRIPTION
**Pull request by julio8a**
**Related Issue:** #1500

## Summary

- Added all sizes to the demo page
- First pass at matching Figma

Working from the [Figma/Tailwind components refactor project](https://github.com/Esri/calcite-components/projects/14)

Project card: https://github.com/Esri/calcite-components/projects/14#card-60667845

### CODE REVIEW

**Comment by bstifle** [comment]
looking pretty good!

Couple things:

1. The horizontal groups need to be adjusted to have some space between, see Figma
2. The vertical groups space between needs to match Figma
3. The numbers should be avenir medium
4. The bars on Error states have a hover color change... which i think is nice. Would like to adjust those, and also add hovers to the blue ones. Red should be ui-danger(idle) and ui-danger-hover(hover). Then ui-brand(idle) and ui-brand-hover(hover)
5. The vertical stacked color bars need to match 4^^^ (currently they are doing their own thing)

**Comment by julio8a** [comment]
These should be good for another review.

**Review by bstifle — APPROVED** [review]
lookin goood

**Comment by julio8a** [comment]
Yay! @jcfranco or @driskull can you give this a quick look?

**Review by jcfranco — APPROVED** [review]
LGTM!

File: src/demos/calcite-stepper.html [hunk 1]
```diff
@@ -19,7 +19,7 @@
     <script src="./_assets/head.js"></script>
   </head>
 
-  <body>
+  <body style="background: white">
```
[the remark this answers is not in the document]
Comment (julio8a) [reply src/demos/calcite-stepper.html:22]: don't think so, I'll remove and MERGE!

[the remark this answers is not in the document]
Comment (julio8a) [reply src/demos/calcite-stepper.html]: updated this in the latest commit.

**Pull request merged**